diff --git a/node_modules/bitcore-lib/lib/crypto/hash.js b/node_modules/bitcore-lib/lib/crypto/hash.js
index 189e8f3..850b706 100644
--- a/node_modules/bitcore-lib/lib/crypto/hash.js
+++ b/node_modules/bitcore-lib/lib/crypto/hash.js
@@ -1,6 +1,6 @@
 'use strict';
 
-var crypto = require('crypto');
+var crypto = require('expo-crypto');
 var BufferUtil = require('../util/buffer');
 var $ = require('../util/preconditions');
 
diff --git a/node_modules/bitcore-lib/lib/crypto/random.js b/node_modules/bitcore-lib/lib/crypto/random.js
index 9938b37..6f57304 100644
--- a/node_modules/bitcore-lib/lib/crypto/random.js
+++ b/node_modules/bitcore-lib/lib/crypto/random.js
@@ -12,7 +12,7 @@ Random.getRandomBuffer = function(size) {
 };
 
 Random.getRandomBufferNode = function(size) {
-  var crypto = require('crypto');
+  var crypto = require('expo-crypto');
   return crypto.randomBytes(size);
 };
 
