diff --git a/node_modules/bitcoinjs-lib/src/payments/embed.js b/node_modules/bitcoinjs-lib/src/payments/embed.js
index 4b7218f..124944e 100644
--- a/node_modules/bitcoinjs-lib/src/payments/embed.js
+++ b/node_modules/bitcoinjs-lib/src/payments/embed.js
@@ -1,5 +1,6 @@
 'use strict';
 Object.defineProperty(exports, '__esModule', { value: true });
+global.Buffer = Buffer ;
 exports.p2data = void 0;
 const networks_1 = require('../networks');
 const bscript = require('../script');
diff --git a/node_modules/bitcoinjs-lib/src/payments/index.js b/node_modules/bitcoinjs-lib/src/payments/index.js
index f820dde..524c9be 100644
--- a/node_modules/bitcoinjs-lib/src/payments/index.js
+++ b/node_modules/bitcoinjs-lib/src/payments/index.js
@@ -1,5 +1,6 @@
 'use strict';
 Object.defineProperty(exports, '__esModule', { value: true });
+global.Buffer = Buffer;
 exports.p2tr =
   exports.p2wsh =
   exports.p2wpkh =
diff --git a/node_modules/bitcoinjs-lib/src/script.js b/node_modules/bitcoinjs-lib/src/script.js
index 6ed7ba2..2eab03f 100644
--- a/node_modules/bitcoinjs-lib/src/script.js
+++ b/node_modules/bitcoinjs-lib/src/script.js
@@ -1,5 +1,6 @@
 'use strict';
 Object.defineProperty(exports, '__esModule', { value: true });
+global.Buffer = Buffer ;
 exports.signature =
   exports.number =
   exports.isCanonicalScriptSignature =
diff --git a/node_modules/bitcoinjs-lib/src/script_signature.js b/node_modules/bitcoinjs-lib/src/script_signature.js
index 638e5f2..1b35422 100644
--- a/node_modules/bitcoinjs-lib/src/script_signature.js
+++ b/node_modules/bitcoinjs-lib/src/script_signature.js
@@ -1,6 +1,7 @@
 'use strict';
 Object.defineProperty(exports, '__esModule', { value: true });
 exports.encode = exports.decode = void 0;
+global.Buffer = Buffer ;
 const bip66 = require('./bip66');
 const types = require('./types');
 const { typeforce } = types;
